examples/vector_databases/tair/Getting_started_with_Tair_and_OpenAI.ipynb

```diff
@@ -0,0 +1,442 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-02-16T12:05:05.718972Z",
+     "start_time": "2023-02-16T12:04:30.434820Z"
+    },
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "! pip install openai tair pandas wget"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prepare your OpenAI API key\n",
+    "\n",
+    "The OpenAI API key is used for vectorization of the documents and queries.\n",
+    "\n",
+    "If you don't have an OpenAI API key, you can get one from [https://beta.openai.com/account/api-keys](https://beta.openai.com/account/api-keys).\n",
+    "\n",
+    "Once you get your key, please add it to your environment variables as `OPENAI_API_KEY`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! export OPENAI_API_KEY=\"your API key\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-02-16T12:05:05.730338Z",
+     "start_time": "2023-02-16T12:05:05.723351Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Test that your OpenAI API key is correctly set as an environment variable\n",
+    "# Note. if you run this notebook locally, you will need to reload your terminal and the notebook for the env variables to be live.\n",
+    "import os\n",
+    "\n",
+    "if os.getenv(\"OPENAI_API_KEY\") is not None:\n",
+    "    print(\"OPENAI_API_KEY is ready\")\n",
+    "else:\n",
+    "    print(\"OPENAI_API_KEY environment variable not found\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We might validate if the server was launched successfully by running a simple curl command:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! curl https://api.openai.com/v1/chat/completions \\\n",
+    "  -H \"Content-Type: application/json\" \\\n",
+    "  -H \"Authorization: Bearer $OPENAI_API_KEY\" \\\n",
+    "  -d '{\"model\": \"gpt-3.5-turbo\",\"messages\": [{\"role\": \"user\", \"content\": \"Say this is a test!\"}]}'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to Tair\n",
+    "First add it to your environment variables.\n",
+    "\n",
+    "Connecting to a running instance of Tair server is easy with the official Python library."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The format of url: redis://[[username]:[password]]@localhost:6379/0\n",
+    "! export TAIR_URL=\"your_tair_url\""
```

Comment:
you might also want to query for this url with `getpass`

Reply:
We have updated the input of api key by getpass in the latest commit.